src/main/java/frc/robot/subsystems/Aiming.java

```diff
@@ -20,19 +18,19 @@
 
 public class Aiming extends SubsystemBase {
 
-    private final SwerveDrive m_drive;
+    //private final SwerveDrive m_drive; this is seemingly not used so I'm going to comment it out.
```

Comment:
Delete all the unused stuff in this file. Also, make sure you get the arguments to the constructor that will become unused